### ISSUE TITLE
Mobile device top category bug

### ISSUE THREAD

**Issue by para2022** [WWBN/AVideo#6952]
Hi Daniel,

Their is a pre-existing bug on the mobile device. The top categories row activated through plugin/layout which shows the categories at the top of the page is broken. It works fine on PC but not on mobile. I’ve attached a screenshot. Thanks.
![99BBE7D2-B6D9-438D-8ABD-D7E6E8E77716](https://user-images.githubusercontent.com/107706817/182297126-34ecfa0f-fd6e-4169-bb16-34fe2f98dde2.jpeg)

**Comment by para2022** [comment]
Any thoughts on this?

**Comment by DanielnetoDotCom** [comment]
Sorry but this is not happening on my side.

If this is really bugging you, for now just remove the top category menu, until we can reproduce and find a solution.

**Comment by para2022** [comment]
ok thank you

**Comment by para2022** [comment]
I have worked out where this problem is coming from Daniel if its any help. If i start a new site and create some categories and turn on the top categories and go to my iphone the top categories are there and it works fine. If i add lots of videos eventually it will stop working. This correlates with joshwhos website as the problem also exists on his site if you view it on a phone and the same happens on my main site. So it is a problem with the videos table thats causing it to fail. Im not sure how this would happen but it is defo the videos table. I tried importing my videos table into a new site and after that the top categories are broken on mobile devices. I undo the table import the top categories work again on mobile devices. Hope this helps.
